x-pack/plugins/security_solution/public/management/cypress/fixtures/exception_list_entries.ts

```diff
@@ -0,0 +1,137 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { ENDPOINT_ARTIFACT_LISTS } from '@kbn/securitysolution-list-constants';
+export const getHieListResponse = (isEmpty = true) => {
+  const data: Array<Record<string, unknown>> = [];
+
+  if (!isEmpty) {
+    data.push({
```

Comment:
Suggestion: maybe use our Data generator to create this?

Reply:
So this stubbing is done only for UI tests for `host isolation exceptions` page, where I wanted to test the page (list item) visibility based on license (and if an entry exists). Doing an actual request is also fine with me. I'll update it as you suggest.

Reply:
Yeah true. This is a duplicate. I think I forgot to remove this. I probably started with this file to do API tests only and then I created a different API test file instead.

Reply:
0a4dcabf9f9e7dccf28ea37f2d946275ed27c8af